pysilcam/silcamgui/guicalcs.py

```diff
@@ -246,9 +245,8 @@ def annotate(datadir, filename):
 
 
 def silcview(datadir):
-    files = [os.path.join(datadir, f) for f in
-            sorted(os.listdir(datadir))
-            if f.endswith('.silc') or f.endswith('.bmp')]
+    files = [os.path.join(datadir, f) for f in sorted(os.listdir(datadir))
+            if f.endswith('.silc') or f.endswith('.bmp') or f.endswith('.silc_mono')]
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->